spack/packages/dla-future/package.py

```diff
@@ -45,6 +45,13 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
         description="Build C API compatible with ScaLAPACK",
     )
 
+    variant(
+        "mpi-gpu-support",
+        default=False,
+        when="@master",
```

Comment:
Probably worth making it conditional also on `+cuda` (x)or `+rocm`?

Reply:
Ah yes, good catch. I'm not aware of another way than conflict either so I'll do that.